indico/modules/events/registration/templates/management/reglist_filter.html

```diff
@@ -12,10 +12,12 @@
             </span>
             {% if has_dropdown %}
                 <ul class="dropdown" style="display: none">
-                    {% if child_item.input_type in ['radio', 'country'] %}
+                    {% if child_item.input_type == 'radio' %}
                         {% set options = child_item.versioned_radioitems.iteritems() %}
                     {% elif child_item.input_type in ['bool', 'checkbox'] %}
                         {% set options = child_item.field_impl.friendly_data_mapping.iteritems() %}
+                    {% elif child_item.input_type == 'country' %}
+                        {% set options = child_item.field_impl.wtf_field_kwargs['choices'] %}
```

Comment:
I wonder if it wouldn't be cleaner to add a `filter_choices` property to those field classes (default return value would be `None`). Then you could just use this in here and wouldn't even need `has_dropdown` (instead you'd check if the filter choices are not none).